packages/fast-tooling/src/data-utilities/relocate.ts

```diff
@@ -3,40 +3,34 @@
  * using data locations (lodash path syntax).
  */
 
-import { at, cloneDeep, get, isFunction, set, unset } from "lodash-es";
+import { cloneDeep, get, isFunction, set, unset } from "lodash-es";
 import { normalizeDataLocationToDotNotation } from "./location";
 import { DataType } from "./types";
 
-export enum TargetPosition {
-    prepend,
-    append,
-    insert,
-}
-
-export interface UpdateDataWithSourceConfig {
+interface UpdateDataWithSourceConfig {
```

Comment:
You don't need to export these?

Reply:
The type should be inferred instead of forcibly cast, you can see its use in the `message-system.utilities.ts` file on line 174. It's a good point though, I'll have to keep an eye out to see if there are requests for it, I'm trying to limit the amount of exports that are available for brevity.

Reply:
I thought it odd that there are functions exported that use params of types that are not also exported.